app/views/themes/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= simple_form_for(@theme) do |f| %>
+  <%= f.input :name %>
+  <%= f.input :description %>
+  <%= f.input :price_cent %>
+  <%= f.input :rating %>
```

Comment:
'rating' should not be created by renter.  Better not be included in the form for creating theme